### ISSUE TITLE
Sorting with relationship doesn't work !

### ISSUE THREAD

**Issue by ifathurrohman** [protonemedia/laravel-splade#454]
- Laravel Version: 10.0
- PHP Version:8.1
- Splade JS Version (npm): 1.4.14
- Splade PHP Version (composer): 1.4.14
- Dev environment (OS, Sail/Valet/etc): ...

### Description:

After upgrade package kirschbaum-development/eloquent-power-joins  to 3.1.1 version sorting with relationship doesn't work !

**Comment by J87NL** [comment]
Same for me, fixed for now by requiring v3.0.0:

```json
        "kirschbaum-development/eloquent-power-joins": "3.0.0",
```

This issue probably isn't Splade-related but should be reported to https://github.com/kirschbaum-development/eloquent-power-joins

**Comment by pascalbaljet** [comment]
Fixed in v1.4.18
